website/docs/cli/private_registry/index.mdx

```diff
@@ -0,0 +1,23 @@
+---
+description: >-
+  Documentation listing third party projects that implement private registries 
+  and their corresponding guides for setting them up.
+---
+
+# Private Registries
+
+OpenTofu, by default, uses the public registry at [registry.opentofu.org](https://opentofu.org/registry/). 
+However, organizations may have a need for a private registry for modules, providers, or policies that need to be kept private.
```

Comment:
I dont think policies are relevant for opentofu

Reply:
Yeah, dont remember why I added that. Have made the requested change.